client/lib/support/index.js

```diff
@@ -0,0 +1,58 @@
+import keyMirror from 'key-mirror';
+
+const root = 'https://support.wordpress.com';
+
+export default keyMirror( {
```

Comment:
I know it's a bit of a PITA (at least in the beginning), but I think that this list should be in alphabetical order - it's easier to skim through it and it's easy to figure out where to put a new link if needed.